sojida/notepads
# ft-getNoteById# Build out GET REST API Endpoint(CRUD) to get a single note

#### What does this PR do?

This PR create an endpoint for a user to get a single note by an id.

#### Description of tasks to be completed

    * Add an endpoint for a GET request 

    * add unit tests


#### How should this be manually tested

    * Clone the repo: Run git clone https://github.com/sojida/notepads/ in the folder where you want the repo to be saved.

    * Navigate into the newly created folder and install the dependencies: cd folder_name && npm install
    
    * Ensure that postman is installed in your laptop. If its not install check the README.

    * Start the server using npm run start:dev

    * Make a GET request to http://localhost:6000/api/v1/notes/7139d3af-b8b4-44f6-a49f-9305791700f4 on your postman and hit the enter button

    * You get a Json object listing the details of that particular note

#### Screenshot
![UNADJUSTEDNONRAW_thumb_48](https://user-images.githubusercontent.com/33469121/54355091-82616c80-4658-11e9-8e86-c4d61896ecd6.jpg)
